src/subscript/sector2fluxnum/sector2fluxnum.py

```diff
@@ -0,0 +1,242 @@
+import argparse
+import os
+import datetime
+from cwrap import open
+from ecl.grid import EclGrid
+from ecl.eclfile import EclFile, FortIO
+from subscript.sector2fluxnum import flux_obj
+from subscript.sector2fluxnum import fluxfile_obj
+from subscript.sector2fluxnum import datafile_obj
+from subscript.sector2fluxnum import completions
+
+
+DESCRIPTION = """
+The script sector2fluxnum will execute different steps in a workflow
+for generating sector models in ECLIPSE and utilize the FLUX
+functionality in ECL100. The workflow will take user defined region of interest (ROI)
+and convert that into a FLUXNUM region that is possible for ECLIPSE to include.
+This includes crossing well trajectories.
+
+Futhermore, the script will generate a template FLUX file and populate this
+with restart data from the RESTART file. This way it is not necessary to run
+a full DUMPFLUX run if the ROI is changed.
+
+The sector models have the same block resolution as the mothe full-field model,
+but opens up for including LGR functionality.
+
+Further details about usage and howto can be found here:
+https://wiki.equinor.com/wiki/index.php/Res:Workflow_for_boundary_condition_handling
+"""
+
+EOL_UNIX = r"\n"
+EOL_WINDOWS = r"\r\n"
+EOL_MAC = r"\r"
+
+EPILOG = ""
+
+
+def get_parser():
+    """Function to create the argument parser that is going to be served to the user.
+
+    Returns:
+    parser (argparse.ArgumentParser): The argument parser to be served
+
+    """
+
+    parser = argparse.ArgumentParser(prog="sector2fluxnum.py", description=DESCRIPTION)
+    parser.add_argument("ECLIPSE_CASE", type=str, help="Eclipse input case")
+    parser.add_argument("OUTPUT_CASE", type=str, help="Name of output FLUX file")
+    parser.add_argument("-i", type=str, help="Example: -i 5-20")
+    parser.add_argument("-j", type=str, help="Example: -j 5-20")
+    parser.add_argument("-k", type=str, help="Example: -k 5-20")
+    parser.add_argument("-f", "--fipnum", type=str, help="Example: -f 46")
+    parser.add_argument(
+        "-r", "--restart", type=str, help="Restart file read separately"
+    )
+    parser.add_argument("-e", "--egrid", type=str, help="EGRID file read separately")
+    parser.add_argument(
+        "-w", "--well", action="store_true", help="Well data read from RST file"
+    )
+    parser.add_argument("--fipfile", type=str, help="FIPNUM file read separately")
```

Comment:
Maybe make more precise that this is the filename of a "fipnum" file that is to be read separately (otherwise the text could indicate that this is a boolean flag).

Reply:
Agree. Could it be something like "Name of FIPNUM file to be read separately for input to region of interest"?